src/components/fileUpload.jsx

```diff
@@ -1,11 +1,32 @@
-import React from "react";
+import React , { useState}  from "react";
 import Button from "@mui/material/Button";
+import { Storage } from 'aws-amplify';
 
 export default function FileUpload() {
```

Comment:
Notice the function name is in Pascal Case. The filname should be in Pascal case as well. So change name fileUpload.jsx to FileUpload.jsx